app/src/scenes/SubCategories/styles.module.css

```diff
@@ -1,11 +1,21 @@
 .mainContent {
-    padding-left: 40px;
+    padding-left: 10px;
     padding-top: 20px;
+    justify-content: center;
 }
 
 .menuItem{
     text-align: center;
     margin-left: 30px;
     margin-right: 30px;
-    font-size: 20px;
+    font-size: 15px;
+}
+
+.pageTitle{
+    margin-top: 14px;
+}
+
+.pageHeader :global .anticon{
```

Comment:
Is it anticon or ant-icon? @Gravewalker666

Reply:
Its anticon. I checked.